README.md

```diff
@@ -46,24 +47,41 @@ Please read [bezier_library/doc/README.md](bezier_library/doc/README.md).
 Quick help
 ----------
 
-**Build**
+**Build using [wstool](http://wiki.ros.org/wstool) and [catkin_tools](http://catkin-tools.readthedocs.io/en/latest/)**
 
-Clone this repository into your catkin source folder and build the workspace:
+Install wstool in order manage the repos inside the workspace
 ```bash
-cd $(catkin_workspace)/src
-git clone https://github.com/ros-industrial-consortium/bezier.git
+sudo apt-get install python-wstool
 ```
 
+```bash
+mkdir catkin_workspace
+mkdir -p catkin_workspace/src
```

Comment:
`mkdir -p ..` creates the argument and all intermediate directories as well - as far as they don't exist. So a single `mkdir -p catkin_workspace/src` is necessary.